.storybook/preview-head.html

```diff
@@ -0,0 +1,22 @@
+<style>
+    html {
+        font-size: 62.5%;
+        background: white;
+    }
+    
+    @font-face{font-family:"Guardian Egyptian Web";src:url('/fonts/egyptiantext-regular.otf');}
+    @font-face{font-family:"Guardian Egyptian Web";font-weight:500;src:url('/fonts/egyptiantext-medium.otf');}
+    @font-face{font-family:"Guardian Egyptian Web";font-weight:bold;src:url('/fonts/egyptiantext-medium.otf');}
+    @font-face{font-family:"Guardian Text Sans Web";src:url('/fonts/GuardianSansWeb-Regular.ttf');}
+    @font-face{font-family:"Guardian Headline";font-weight:200;src:url('/fonts/GHGuardianHeadline-Light.ttf');}
+    @font-face{font-family:"Guardian Headline";font-weight:400;src:url('/fonts/GHGuardianHeadline-Regular.ttf');}
+    @font-face{font-family:"Guardian Headline";font-weight:500;src:url('/fonts/GHGuardianHeadline-Medium.ttf');}
+    @font-face{font-family:"Guardian Headline";font-weight:700;src:url('/fonts/GHGuardianHeadline-Bold.ttf');}
+    @font-face{font-family:"Guardian Icons";src:url('/fonts/icons.otf');}
+    body{margin:0;font-family:'Guardian Egyptian Web';font-size:1.6em;line-height:1.5em;overflow-x:hidden;}
+    figure{margin:1em 0;}
+    figure.element-embed{margin:2em 0;}
+    .js-email-sub__iframe{width:100%;}
+    .js-email-sub__iframe + figcaption{margin-top:-8px;}
+    video, .element-atom iframe, .element-audio iframe{width:100%;}
+</style>
```

Comment:
Does this differ from the styles in `page.tsx`?

Reply:
Nope, but I couldn't think of a better way of including these global styles in the storybook components...